lib/mix/tasks/create_dummy_accounts.ex

```diff
@@ -1,313 +1,315 @@
-          zip_code: random_zip_code(),
-          language: "DE-de",
-          legal_terms_accepted: true,
-          gender: gender,
-          height: height,
-          mobile_phone: random_mobile_phone_number(),
-          birthday: birthday,
-          hashed_password: Faker.UUID.v4(),
-          occupation: Person.En.title_descriptor(),
-          minimum_partner_height: height - 30,
-          maximum_partner_height: height + 15,
-          minimum_partner_age: minimum_partner_age(age),
-          maximum_partner_age: age + 10,
-          partner_gender: opposite_gender(gender),
-          search_range: hd(Enum.take_random([5, 10, 20, 50, 100, 250], 1))
-        })
-
-      # create profile photo
-      Photo.create!(Map.merge(photo, %{user_id: user.id}))
-
-      # create about me story
-      story =
-        Story.create!(%{
-          headline_id: about_me_headline,
-          user_id: user.id,
-          content: random_lorem_ipsum(),
-          position: 1
-        })
+if Mix.env() == :dev || Mix.env() == :test do
+  defmodule Mix.Tasks.CreateDummyAccounts do
+    @moduledoc """
+    This task creates dummy accounts for development.
+    """
+
+    use Mix.Task
+    alias Animina.Accounts.Photo
+    alias Animina.Accounts.User
+    alias Animina.Narratives
+    alias Animina.Narratives.Headline
+    alias Animina.Narratives.Story
+    alias Animina.Traits.Flag
+    alias Animina.Traits.UserFlags
+    alias Faker.Person
+    alias Faker.Phone
+
+    require Ash.Query
+
+    def run(args) do
+      Mix.Task.run("app.start", [])
+
+      (List.first(args) || "100")
+      |> String.to_integer()
+      |> generate_users()
+
+      User.read!()
+      |> print_table()
+    end
 
-      # create about me story photo
-      Photo.create!(Map.merge(photo, %{user_id: user.id, story_id: story.id}))
+    def generate_users(0) do
+    end
 
-      # create random stories
-      Enum.each(2..Enum.random(2..8), fn i ->
-        create_random_story(user, i)
+    def generate_users(counter) do
+      about_me_headline = get_about_me_headline()
+
+      1..counter
+      |> Enum.each(fn _ ->
+        height = Enum.take_random(Enum.to_list(150..210), 1) |> hd
+        birthday = Faker.Date.date_of_birth()
+        age = (Date.diff(Date.utc_today(), birthday) / 365) |> round
+        gender = Enum.take_random(["male", "female"], 1) |> hd
+        photo = random_photo_url(gender) |> download_photo("#{Faker.UUID.v4()}.png")
+
+        user =
+          User.create!(%{
+            email: Faker.Internet.email(),
+            username: Faker.Internet.user_name() |> String.slice(0..14),
+            name: Faker.Person.name(),
+            zip_code: random_zip_code(),
+            language: "DE-de",
+            legal_terms_accepted: true,
+            gender: gender,
+            height: height,
+            mobile_phone: random_mobile_phone_number(),
+            birthday: birthday,
+            hashed_password: "test",
```

Comment:
How can the value of the `hashed_password` be `test` if I want the password to be test?

Reply:
Thanks for this oversight , I looked at ash_authentication and noticed they use Bcrypt so we now have
 ` Bcrypt.hash_pwd_salt("test")` , I have tested this locally and it works